tests/test_admin.py

```diff
@@ -2408,3 +2408,119 @@ def test_change_view_action_compare_versions_three_selected(self):
             response = self.client.post(endpoint, data, follow=True)
 
         self.assertContains(response, "Exactly two versions need to be selected.")
+
+
+class ExtendedVersionAdminTestCase(CMSTestCase):
+
+    def test_extended_version_change_list_display_renders_from_provided_list_display(self):
+        """
+        Check the list_display options added by ExtendedVersionAdminTestCase
```

Comment:
All fields are present for a content object if the class inheriting the mixin: ExtendedVersionAdminMixin has set any fields to display. This will be the list of fields the user has added and the fields & actions set by the mixin.